eras/conway/impl/src/Cardano/Ledger/Conway/Genesis.hs

```diff
@@ -45,17 +47,11 @@ instance Crypto c => NoThunks (ConwayGenesis c)
 
 -- | Genesis are always encoded with the version of era they are defined in.
 instance Crypto c => DecCBOR (ConwayGenesis c) where
-  decCBOR =
-    ConwayGenesis
-      <$> decCBOR
-      <*> decCBOR
-      <*> decCBOR
+  decCBOR = decode $ RecD ConwayGenesis <! From <! From <! From
 
-instance Crypto c => EncCBOR (ConwayGenesis c) where
+instance forall c. Crypto c => EncCBOR (ConwayGenesis c) where
```

Comment:
`forall` in class instance header is not needed for `ScopedTypeVariables`, all type variables are readily available when this extension is turned on. `forall` is only needed for function type signatures.
```suggestion
instance Crypto c => EncCBOR (ConwayGenesis c) where
```

Reply:
TIL. Thanks! :heart: